archive-peoplegroups.php

```diff
@@ -0,0 +1,106 @@
+<?php declare(strict_types=1); ?>
+<?php get_header(); ?>
+
+<?php dt_print_breadcrumbs( null, __( "People Groups" ) ); ?>
+
+<div id="content">
+    
+    <div id="inner-content" class="grid-x grid-margin-x">
+        
+        <main id="main" class="large-8 medium-8 cell" role="main">
+            
+            <?php
+            $args = array(
+                'post_type' => 'peoplegroups',
+            
+            );
+            $query1 = new WP_Query( $args );
+            ?>
+            <?php if ( $query1->have_posts() ) : while ( $query1->have_posts() ) : $query1->the_post(); ?>
+                
+                <!-- To see additional archive styles, visit the /parts directory -->
+                <?php get_template_part( 'parts/loop', 'prayer' ); ?>
+            
+            <?php endwhile; ?>
+                
+                <?php disciple_tools_page_navi(); ?>
+            
+            <?php else : ?>
+                
+                <?php get_template_part( 'parts/content', 'missing' ); ?>
+            
+            <?php endif; ?>
+        
+        </main> <!-- end #main -->
+        
+        <aside class="large-4 medium-4 cell">
+            
+            <section class="bordered-box">
+                
+                <?php include 'searchform.php'; ?>
+            
+            </section>
+            
+            <section class="bordered-box">
+                
+                <h4>Recent Posts</h4>
+                
+                <?php $args = array(
+                    'numberposts' => 10,
+                    'offset' => 0,
+                    'category' => 0,
+                    'orderby' => 'post_date',
+                    'order' => 'DESC',
+                    'include' => '',
+                    'exclude' => '',
+                    'meta_key' => '',
+                    'meta_value' =>'',
+                    'post_type' => 'peoplegroups',
+                    'post_status' => 'draft, publish, future, pending, private',
+                    'suppress_filters' => true
+                );
+                
+                $recent_posts = wp_get_recent_posts( $args, ARRAY_A );
+                
+                echo '<ul>';
+                ?>
+                <?php foreach ($recent_posts as $recent_post): ?>
+                    <li><a href="<?php echo $recent_post['guid'] ?>"><?php echo $recent_post['post_title'] ?></a></li>
+                <?php endforeach; ?>
```

Comment:
the guid should not be used as a link to the post. 
https://developer.wordpress.org/reference/functions/the_guid/
try get_permalink() instead

Reply:
Actually, that was too much of a copy paste. I'll clean it up.